mailscanner/lib/pear/Pager/Common.php

```diff
@@ -1627,7 +1627,7 @@ public function setOptions($options)
         $this->_perPage = max($this->_perPage, 1); //avoid possible user errors
 
         if ($this->_useSessions && !isset($_SESSION)) {
-            session_start();
+           // session_start();
```

Comment:
I left it in but commented as it's a pear package, just incase we import an update version of the package.

There should be no case where that condition will ever occur as the session is started long before we use the pear package.